wallet-new/server/Main.hs

```diff
@@ -57,22 +58,17 @@ import qualified Cardano.Wallet.WalletLayer.Kernel as WalletLayer.Kernel
 defaultLoggerName :: LoggerName
 defaultLoggerName = "node"
 
-{-
-   Most of the code below has been copied & adapted from wallet/node/Main.hs as a path
-   of least resistance to make the wallet-new prototype independent (to an extent)
-   from breaking changes to the current wallet.
--}
-
--- | The "workhorse" responsible for starting a Cardano edge node plus a number of extra plugins.
-actionWithWallet :: (HasConfigurations, HasCompileInfo)
+-- | The legacy function responsible for starting a Cardano edge node plus a
+-- number of extra plugins.
+actionWithLegacyWallet :: (HasConfigurations, HasCompileInfo)
```

Comment:
Starting to pave the way to the final migration: let's start rebranding legacy things properly.